lib/has_friendship/friendable.rb

```diff
@@ -88,6 +88,12 @@ def friends_with?(friend)
         HasFriendship::Friendship.find_relation(self, friend, status: 2).any?
       end
 
+	  def get_invitation_request(friend)
```

Comment:
Don’t prefix method names with “get”.

Reply:
hey,
i'll make some changes. I don't know what happened with the title!